### PR TITLE
Fixed Typos

### PR DESCRIPTION
**Pull request by engnadeau**
- Used https://github.com/client9/misspell to identify and fix typos throughout the repo
- Thanks for all the hard work!

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tensorflow/18806)

### CODE REVIEW

File: tensorflow/examples/udacity/6_lstm.ipynb [hunk 1]
```diff
@@ -789,7 +789,7 @@
             " 2000 : 1.69528644681 learning rate: 10.0\n",
             "Minibatch perplexity: 5.13\n",
             "================================================================================\n",
-            "vers soqually have one five landwing to docial page kagan lower with ther batern\n",
+            "vers soqually have one five landwing to docial page kagan lower with there batern\n",
```
Comment (ekelsen) [inline tensorflow/examples/udacity/6_lstm.ipynb]: It's not clear that this should be changed.

**Comment by ekelsen** [comment]
LGTM, except for the one case where I think the suggestion should be reverted.

**Comment by nnadeau** [comment]
@ekelsen https://github.com/tensorflow/tensorflow/pull/18806/commits/c86e47c402e7bb347389d4690618ee5f8e11dab7 reverts the overzealous change 😛

File: tensorflow/contrib/autograph/operators/control_flow.py [hunk 2]
```diff
@@ -163,7 +163,7 @@ def while_loop(loop_cond, loop_body, init_state, extra_deps, opts=None):
     Tuple containing the final state.
   """
   # TODO(mdan): Consider adding a generic mechanism for dynamic dispatch.
-  # That could be somethins as simple as a collection of dispatch rules, with
+  # That could be somethings as simple as a collection of dispatch rules, with
```
Comment (av8ramit) [inline tensorflow/contrib/autograph/operators/control_flow.py]: "something"?

**Comment by caisq** [comment]
Ping @nnadeau

**Comment by nnadeau** [comment]
@av8ramit @caisq extra 's' removed in 08a3319

**Review by av8ramit — DISMISSED** [review]

**Review by caisq — DISMISSED** [review]

**Pull request merged**